config/vendor-api-v1.yml

```diff
@@ -746,6 +747,12 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/Qualification"
+        missing_gcses_explanation:
+          type: string
+          nullable: true
+          maxLength: 717 # 700 words: 3 x 200 (one per required GCSE) + 100 for our boilerplate
```

Comment:
The comment is a bit unclear. I think it should say `700 characters:`; but even then it's set to 717 which isn't explained. Should we also round it up so it looks a bit less arbitrary?

This is a nit, and can be ignored.

Reply:
This is me being unnecessarily fussy: I've been trying to stick to powers-of-two elsewhere, so this should really be rounded up to the next available power of 2 (ie 1024). Will fix.

Reply:
I was also going to suggest 1000 or 1024, whichever is rounder 😉